packages/artifact/src/internal/shared/config.ts

```diff
@@ -1,3 +1,9 @@
+// Used for controlling the highWaterMark value of the zip that is being streamed
+// The same value is used as the chunk size that is use during upload to blob storage
+export function getUploadChunkSize(): number {
+  return 8 * 1024 * 1024 // 8 MB Chunks
```

Comment:
We should make sure we use [append blocks ](https://learn.microsoft.com/en-us/azure/storage/blobs/scalability-targets#scale-targets-for-blob-storage)for artifacts, as block blobs are limited to 4GiB.  

Also the chunk size supported by Azure is 4MiB.